unit-tests/run-unit-tests.py

```diff
@@ -597,11 +599,11 @@ def test_wrapper( test, configuration = None ):
             continue
         #
         tags.update( test.config.tags )
-        #
         tests.append( test.name )
-        #
-        if list_tags or list_tests:
+        if list_only:
+            n_tests += 1

```

Comment:
Why do we need to increase this counter?

Reply:
Because it's the counter for the number of tests we found, and without which (if it's 0) then we print an error...